src/tools/perf/lib/libperf.c

```diff
@@ -1746,8 +1750,8 @@ size_t ucx_perf_get_message_size(const ucx_perf_params_t *params)
 unsigned rte_peer_index(unsigned group_size, unsigned group_index)
 {
     int size       = group_size;
-    int index      = group_index;
-    int peer_index = size - 1 - index;
+    int idx        = group_index;
```

Comment:
btw, maybe we can remove size and idx variables? why are they needed?
can cast to int directly or assert(group_index < group_size)